src/v2/view-builder/views/shared/ChallengeAuthenticatorDataFormView.js

```diff
@@ -1,3 +1,4 @@
+// Common view for OV push and custom push.
```

Comment:
Lets rename this file since to something like:
* ChallengeAuthenticatorDataPushOTPFormView.js
* ChallengeAuthenticatorDataOVCustomAppFormView.js

This file is, as you are commenting here, is for 2 authenticators. So I think it will be a good idea to name the file to something along those lines, otherwise if I read the name as "ChallengeAuthenticatorDataFormView" it sounds like it applies to ALL authenticators.